### DISCUSSION TITLE
How many combinations?

### DISCUSSION THREAD

**Discussion by nojedzto** [Discord-OTP-Forcer/Discord-OTP-Forcer#136] [Community Support]
How many possible 8-digit codes are there? I have 100,000 attempts behind me and I wonder how many more?

**Comment by Inbydev** [comment]
@nojedzto could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

**Reply by nojedzto** [reply]
What is it?

**Reply by Inbydev** [reply]
The error that you see, I don't know the language, but if you could try with the latest update from this repository, it may work now!
